scripts/intfutil

```diff
@@ -222,32 +243,40 @@ def po_speed_dict(po_int_dict, appl_db):
         po_speed_dict = {}
         return po_speed_dict
 
-def appl_db_portchannel_status_get(appl_db, po_name, status_type, portchannel_speed_dict):
+def appl_db_portchannel_status_get(appl_db, config_db, po_name, status_type, portchannel_speed_dict):
     """
     Get the port status
     """
     full_table_id = "LAG_TABLE:" + po_name
+    po_table_id = "PORTCHANNEL|" + po_name
     #print(full_table_id)
     if status_type == "speed":
         status = portchannel_speed_dict[po_name]
         return status
     if status_type == "vlan":
         status = "routed"
         return status
+    if status_type == "mtu":
+        status = config_db.get(config_db.CONFIG_DB, po_table_id, status_type)
+        return status
     status = appl_db.get(appl_db.APPL_DB, full_table_id, status_type)
     #print(status)
     if status is None: 
         return "N/A"
     return status 
 
 
+
+
+
+
```

Comment:
6 blank lines is excessive. 2 should be plenty. Please remove 4 blank lines.

Reply:
done